### DISCUSSION TITLE
Display troubles

### DISCUSSION THREAD

**Discussion by grisuat** [nakhonthai/ESP32APRS_T-TWR#43] [General]
Hi,

since the update to 0.4 I have sometimes troubles with the display: After some time it looses it center (see two examples attached). First I thought it is caused by low battery but it also happened today while on USB-C.

Does anyone have same oberservations?

73 de Bernd, OE7BSH

![gestern](https://github.com/user-attachments/assets/857d49cb-aac9-47e8-b42a-93688642fccf)
![heute](https://github.com/user-attachments/assets/afe0cca7-991e-4cf2-8566-908904ba7c6e)

**Comment by sethpark777** [comment]
Yes i get the same behavior on V2.1 hardware. I have not linked it to any particular event yet. 
![image](https://github.com/user-attachments/assets/fe67fadd-b813-4db6-88d2-315def5351cc)
![image](https://github.com/user-attachments/assets/8f70e94c-739b-4cd0-8f61-6e5b02f02ac4)

**Comment by CcRup** [comment]
Same here. TWR Plus 2.0. Mine also flips upside down sometimes. Seems to be random just after TX in Tracker mode. Haven't tried other modes yet.
